x/incentive/types/codec.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+)
+
+var (
+	amino = codec.NewLegacyAmino()
+
+	// ModuleCdc references the global x/incentive module codec. Note, the codec
+	// should ONLY be used in certain instances of tests and for JSON encoding as
+	// Amino is still used for that purpose.
```

Comment:
```suggestion
```

Reply:
this will be also used by all dependencies, we removed all that legacy comments.